I/ipx/bundled/patches/example.patch

```diff
@@ -0,0 +1,30 @@
+--- a/Makefile
++++ b/Makefile
+@@ -7,7 +7,7 @@
+ # The default creates the static and shared library in the lib/ subdirectory and
+ # compiles the examples in the example/ subdirectory.
+ 
+-default: static shared examples
++default: static shared
```

Comment:
Can't you call `make static shared` (or only `make shared`) instead of patching the Makefile?